docs/deep-dive/01-dd-create-inline-function-serverless.md

```diff
@@ -0,0 +1,152 @@
+
+> **TIP:** Read about [Function’s specification](#details-function-s-specification) if you are interested in its signature, `event` and `context` objects, and custom HTTP responses the Function returns.
+
+## Steps
+
+Follow these steps:
+
+<div tabs name="steps" group="create-function">
+  <details>
+  <summary label="cli">
+  Kyma CLI
+  </summary>
+
+1.  Export these variables:
+
+    ```bash
+    export NAME={FUNCTION_NAME}
+    export NAMESPACE={FUNCTION_NAMESPACE}
+    ```
+
+2.  Create your local development workspace.
+
+    a. Create a new folder to keep Function's code and configuration in one place.
+
+    ```bash
+    mkdir {FOLDER_NAME}
+    cd {FOLDER_NAME}
+    ```
+
+    b. Create initial scaffolding for the Function using the dedicated CLI command.
+
+    ```bash
+    kyma init function --name $NAME --namespace $NAMESPACE
+    ```
+
+3.  Code and configure.
+
+    Open the workspace in your favorite IDE. If you have Visual Stdio Code installed, run the following command from the terminal in your workspace folder:
+
+    ```bash
+    code .
+    ```
+
+    It's time to inspect the code and the `config.yaml` file. Feel free to adjust the "Hello World" sample code.
+
+4.  Deploy and verify.
+
+    a. Call the `apply` command from the workspace folder. It will build the container and run it on the Kyma runtime pointed by your current KUBECONFIG file.
+
+      ```bash
+      kyma apply function
+      ```
+
+    b. Check if your Function was created successfully.
+
+      ```bash
+      kubectl get functions $NAME -n $NAMESPACE
+      ```
+
+    You should get a result similar to the following example:
+
+    ```bash
+    NAME            CONFIGURED   BUILT     RUNNING   RUNTIME    VERSION   AGE
+    test-function   True         True      True      nodejs12   1         96s
+    ```
+
+  </details>
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+1.  Export these variables:
+
+    ```bash
+    export NAME={FUNCTION_NAME}
+    export NAMESPACE={FUNCTION_NAMESPACE}
+    ```
+
+2.  Create a Function CR that specifies the Function's logic:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: serverless.kyma-project.io/v1alpha1
+    kind: Function
+    metadata:
+      name: $NAME
+      namespace: $NAMESPACE
+    spec:
+      runtime: nodejs12
+      source: |
+        module.exports = {
+          main: function(event, context) {
+            return 'Hello World!'
+          }
+        }
+    EOF
+    ```
+
+3.  Check if your Function was created successfully and all conditions are set to `True`:
+
+    ```bash
+    kubectl get functions $NAME -n $NAMESPACE
+    ```
+
+    You should get a result similar to the following example:
+
+    ```bash
+    NAME            CONFIGURED   BUILT     RUNNING   RUNTIME    VERSION   AGE
+    test-function   True         True      True      nodejs12   1         96s
+    ```
+
+    </details>
+    <details>
+    <summary label="busola-ui">
+    Busola UI
```

Comment:
AFAIK Busola is the internal project name
@valentinvieriu Should we keep `Console` or switch to `Busola` when referencing UI on the website?

Reply:
Busola it is.